blockstore/src/test/java/brooklyn/location/blockstore/openstack/OpenStackVolumeManagerLiveTest.java

```diff
@@ -20,7 +20,7 @@
 public class OpenStackVolumeManagerLiveTest extends AbstractVolumeManagerLiveTest {
 
     public static final String PROVIDER = "openstack-nova";
-    public static final String ENDPOINT = "https://lon.identity.api.rackspacecloud.com/v2.0/";
+    public static final String ENDPOINT = "https://cloudsoft2-lon.openstack.blueboxgrid.com:5000/v2.0/";
```

Comment:
Unfortunately this is a public project and we cannot include that URL in master.